Gemfile

```diff
@@ -53,3 +53,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# Added at 2018-04-12 18:22:02 -0500 by root:
+gem "coffee-script", "~> 2.4"
```

Comment:
Do we use coffee script at all? I don't see it, and I'd prefer to stick with JQuery if we can.